.github/workflows/rgw-multisite-test/action.yml

```diff
@@ -0,0 +1,94 @@
+    required: true
+  ceph-image:
+    description: Ceph image to use for the workflow (e.g., quay.io/ceph/ceph:v16.2.5)
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - name: setup cluster resources
+      uses: ./.github/workflows/setup-cluster-resources
+      with:
+        github-token: ${{ inputs.github-token }}
+
+    - name: install additional deps for object testing
+      shell: bash --noprofile --norc -eo pipefail -x {0}
+      run: |
+        sudo apt-get install -y s3cmd
+
+    - name: use local disk into two partitions
+      shell: bash --noprofile --norc -eo pipefail -x {0}
+      run: |
+        BLOCK=$(sudo lsblk --paths|awk '/14G/ {print $1}'| head -1)
+        tests/scripts/github-action-helper.sh use_local_disk
+        tests/scripts/create-bluestore-partitions.sh --disk "$BLOCK" --osd-count 2
+        sudo lsblk
+
+    - name: set Ceph version in CephCluster manifest
+      shell: bash --noprofile --norc -eo pipefail -x {0}
+      run: |
+        tests/scripts/github-action-helper.sh replace_ceph_image \
+          "cluster/examples/kubernetes/ceph/cluster-test.yaml" "${{ inputs.ceph-image }}"
+
+    - name: deploy first cluster rook
+      shell: bash --noprofile --norc -eo pipefail -x {0}
+      run: |
+        tests/scripts/github-action-helper.sh deploy_first_rook_cluster
+        kubectl create -f cluster/examples/kubernetes/ceph/object-multisite-test.yaml
+        # wait for multisite store to be created
+        tests/scripts/github-action-helper.sh wait_for_rgw rook-ceph
+
+    - name: prep second cluster pull realm config
+      shell: bash --noprofile --norc -eo pipefail -x {0}
+      run: |
+        cd cluster/examples/kubernetes/ceph/
+        IP_ADDR=$(kubectl -n rook-ceph get svc rook-ceph-rgw-multisite-store -o jsonpath="{.spec.clusterIP}")
+        yq w -i -d1 object-multisite-pull-realm-test.yaml spec.pull.endpoint http://${IP_ADDR}:80
+        BASE64_ACCESS_KEY=$(kubectl -n rook-ceph get secrets realm-a-keys -o jsonpath="{.data.access-key}")
+        BASE64_SECRET_KEY=$(kubectl -n rook-ceph get secrets realm-a-keys -o jsonpath="{.data.secret-key}")
+        sed -i 's/VzFjNFltMVdWRTFJWWxZelZWQT0=/'"$BASE64_ACCESS_KEY"'/g' object-multisite-pull-realm-test.yaml
+        sed -i 's/WVY1MFIxeExkbG84U3pKdlRseEZXVGR3T3k1U1dUSS9KaTFoUVE9PQ==/'"$BASE64_SECRET_KEY"'/g' object-multisite-pull-realm-test.yaml
+
+    - name: deploy second cluster rook
+      shell: bash --noprofile --norc -eo pipefail -x {0}
+      run: |
+        tests/scripts/github-action-helper.sh deploy_second_rook_cluster
+        kubectl create -f cluster/examples/kubernetes/ceph/object-multisite-pull-realm-test.yaml
+        # wait for realms to be pulled and zone-b-multisite-store to be created
+        tests/scripts/github-action-helper.sh wait_for_rgw rook-ceph-secondary
+
+    - name: wait for ceph cluster 1 to be ready
+      shell: bash --noprofile --norc -eo pipefail -x {0}
+      run: |
+        mkdir test
```

Comment:
why create this dir?

Reply:
I think `validate_cluster.sh` used to write data into `test/`. That's probably a carryover from then that I can remove.